Dockerfile.windows

```diff
@@ -1,17 +1,24 @@
-FROM microsoft/dotnet:1.1-sdk-msbuild
+# escape=`
+FROM microsoft/dotnet:2.0-sdk-nanoserver
```

Comment:
I rebuilt. But I still get following error. This error is result of "dotnet run --framework netcoreapp2.0".

```
web_1  | Using launch settings from C:\MusicStore\samples\MusicStore\Properties\launchSettings.json...
web_1  | It was not possible to find any compatible framework version
web_1  | The specified framework 'Microsoft.NETCore.App', version '2.0.0-preview3-25428-01' was not found.
web_1  |   - Check application dependencies and target a framework version installed at:
web_1  |       \
web_1  |   - Alternatively, install the framework version '2.0.0-preview3-25428-01'.
```

full log
https://gist.github.com/takekazuomi/f2c8bcdc64c5b7cf6d68a16e745cadd5